cloudformation/stacks/AutoSpotting/template.json

```diff
@@ -71,6 +71,11 @@
       "Default": "",
       "Description": "Comma separated list of disallowed instance types for spot, in case you want to exclude specific types (also support globs). Example: 't2.*,m4.large'",
       "Type": "String"
+    },
+    "FilterByTags": {
+      "Default": "",
+      "Description": "Comma separated list of tag=value on which to filter the ASGs that autospotting considers.  By default 'spot-enabled=true'.  Example: 'spot-enabled=true,environment=dev'",
+      "Type": "String"
```

Comment:
I'm not au fait with cloudformation, and I do not know where cloud formation uses the "Default", is it just documentation?  

Instead of adding a default, can we say in the decription:  If left blank, the code adds spot-enabled=true when nothing is specified (like it does now)?  

i.e. the is no default of Regions, but it defaults to us-east-1 for example in autospotting.go

I'll update the description, with this, and will await feedback

Reply:
Fine with me!